src/openeo_grass_gis_driver/actinia_processing/base.py

```diff
@@ -62,6 +62,7 @@ class GrassDataType(Enum):
     RASTER = "raster"
     VECTOR = "vector"
     STRDS = "strds"
+    rastercube = "rastercube"
```

Comment:
What is the reason to not call it `STAC` directly? `STRDS` is also kind of a rastercube.
And also I would prefer UPPERCASE for variable name to be consistent with the others.

Reply:
Because of the structure of the name (`stac.<instance_id>.rastercube.<collection_id>`). It would be needed to create a new data type in order to include and read just stac.

Reply:
Ok - but it is just about the value and the variable could be named differently, right? I just think it might be more clear if it would be like `STAC = "rastercube"`. Because then the actinia datatypes are more obvious.